entrypoint.sh

```diff
@@ -1,9 +1,17 @@
 #!/bin/bash
 
+export TARGET_API="https://api.qa.notprod.lev.homeoffice.gov.uk"
```

Comment:
Should this be hard-coded here?
Shouldn't it come from the config file?

Reply:
That's just a default, the config files for other envs set an explicit target

Reply:
It's a bit odd to set that but not anything else.

Would it make more sense to remove it and instead do:
```bash
set -euo pipefail
```

Reply:
I suppose it's not a big deal. Have a think. I'm happy to merge with no changes here.

Reply:
I think because most/all of the rest of the config values are "sensitive" this is the only one where it makes sense to set a default. I think I'm inclined to leave it as is...